scripts/issue/1133/upkeep_manager_integration.py

```diff
@@ -0,0 +1,39 @@
+from brownie import interface
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+"""
+UpKeep references:
```

Comment:
```suggestion
Upkeep references:
```
😂

Reply:
i am dying 😢. sorted in ca4eb2e2660147d016bf24f90a4600e7dce43073